### PR TITLE
Update teams-for-vdi.md

### PR DESCRIPTION
**Pull request by voltarone**
Adding more clarity for the Un-install Scenarios

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-for-vdi.md [hunk 1]
```diff
@@ -152,12 +152,12 @@ To learn more about Teams and Office 365 ProPlus, see [How to exclude Teams from
 
 3. Uninstall the MSI from the VDI VM. 
 
-    There are two ways to uninstall Teams:  
+    There are two ways to uninstall Teams, depending if the deployment is per-machine or per-user:
   
-    - PowerShell script (recommended):
+    - For Per-User Deployment only -- PowerShell script:
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md]:
```suggestion
    - For Per-User Deployment only—PowerShell script:
```

File: Teams/teams-for-vdi.md [hunk 2]
```diff
@@ -152,12 +152,12 @@ To learn more about Teams and Office 365 ProPlus, see [How to exclude Teams from
...
     You can use this [PowerShell script](scripts/powershell-script-teams-deployment-clean-up.md) to clean up Teams from target machines or users. It should be executed for every user on a targeted machine. 
     
-    - Command line:
+    - For Per-Machine or Per-User -- Command line:
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md]:
```suggestion
    - For Per-Machine or Per-User—Command line:
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Thanks for the input, @voltarone. I've followed up with the PMs and they have changes to this section that will supersede your updates. Closing this without merging.  Thanks.

**Pull request closed without merging**